contracts/sol6/Dao/KyberFeeHandler.sol

```diff
@@ -353,6 +354,16 @@ contract KyberFeeHandler is IKyberFeeHandler, Utils5 {
         emit BurnConfigSet(_sanityRate, _weiToBurn);
     }
 
+    /// @dev Allow to set network proxy address by using burnConfigSetter
+    /// @param _newProxy new Kyber Proxy contract
+    function setNetworkProxy(IKyberProxy _newProxy) external onlyBurnConfigSetter {
```

Comment:
We need a new name, since burnConfigSetter is able to set more than the burn configuration now....

Reply:
yes
right.
any suggestion?
I don't have at the moment

Reply:
Maybe we should consider calling them (all different setter in the DAO scope) as "DaoOperator" and make them consistent to be one. We will also have detailed documentation on the Dao Operator as well.

Reply:
so we have one DaoOperator contract that is inherited by KyberDAO and KyberFeeHandler?

if yes sounds good to me.

Reply:
Yes that was what I meant. Having only one DaoOperator and Kyber team controls it at the beginning